swagger_parser/lib/src/parser/parser.dart

```diff
@@ -595,7 +593,7 @@ class OpenApiParser {
           jsonKey: name,
           defaultValue: arrayType.type.defaultValue,
           isRequired: isRequired,
-          nullable: arrayType.type.nullable,
+          nullable: map[_nullableConst] == true,
```

Comment:
It is more correct to do so: 
map[_nullableConst]?.toString() .toBool()

Reply:
updated